src/pages/Index.vue

```diff
@@ -214,6 +214,9 @@ query {
     padding-top: 100px;
     border: 4px solid black;
 }
+.header + .main-content {
+    margin-top: 40px;
```

Comment:
@NickSto We should probably stick with rem here for margins?

Reply:
Done in #807